src/generate/index.ts

```diff
@@ -8,3 +8,6 @@ export { Api } from "./api";
 export { ApiGroup } from "./apiGroup";
 export { ApiCollection } from "./apiCollection";
 export { Template } from "./template";
+
+import * as handlebarsAmfHelpers from "./handlebarsAmfHelpers";
+export { handlebarsAmfHelpers };
```

Comment:
I'm curious if we should we should also or instead this include a subpath export? https://nodejs.org/api/esm.html#esm_subpath_exports

Reply:
@bredmond-sf I didn't exactly use subpath-exports but imported all the functions we want to export in index.ts and exporting everything there. Does it satisfy the concern you had or do you still want me to look into using subpath-exports?